package.json

```diff
@@ -118,6 +120,7 @@
     "@types/tar": "4.0.0",
     "@types/through": "0.0.29",
     "@types/yargs": "11.1.0",
+    "all-contributors-cli": "^5.2.1",
```

Comment:
Seems like the only dep which isn't pinned ^^

Reply:
You're right. Thank you for pointing that out. Changing that now.